.github/dependabot.yml

```diff
@@ -4,6 +4,7 @@ updates:
     directory: "/npm"
     schedule:
       interval: "daily"
+    versioning-strategy: increase
```

Comment:
Do we need to do this in any of the other projects?

Reply:
Yep. I've already done the other ones

Reply:
👍